internal/pkg/cli/cmd/template/test.go

```diff
@@ -41,8 +41,14 @@ func TestCommand(p dependencies.Provider) *cobra.Command {
 				return err
 			}
 
+			// Options
+			options := testOp.Options{
+				LocalTestsOnly: d.Options().GetBool("local-test"),
```

Comment:
Mozno by som tu flag nazval skor `--only-local` a pridal rovno aj `--only-remote`.

Reply:
Lebo to, ze sa jedna o test, je zrejme z nazvu prikazu.

Reply:
Jo, díky. 👍 Jsem s tím bojoval, to `only` mi tam pořád chybělo ale nechtěl jsem tam mít tři slova 😅

Reply:
jen drobnost, pojmenoval bych to spíš naopak `--local-only` a `remote-only`. Ve slovosledu se to dává spíš až za to slovo (a i třeba `git diff` má `--name-only`)

Reply:
:+1: